### DISCUSSION TITLE
brew upgrade

### DISCUSSION THREAD

**Discussion by carlca** [Homebrew/discussions#2149] [Everyday usage]
Don't do it if you're in a hurry to do something soon afterwards. It's still going after an hour and seems to have downloaded the entire internet. Twice 😉

**Comment by SMillerDev** [comment]
A better idea is to do it often, that way it'll be much faster if you ever need something from brew (and you won't be using software with security issues). Personally I run it every day when I open my computer to work, and it rarely downloads more than 3 packages because the rest is updated on the other days.
